modules/dists/SparseBlockDist.chpl

```diff
@@ -117,6 +116,64 @@ class SparseBlockDom: BaseSparseDom {
     dsiAdd((ind,));
   }
 
+  proc dsiBulkAdd(inds: [] index(rank, idxType), isSorted=false, isUnique=false,
+      preserveInds=true){
+  
+    if !isSorted && preserveInds {
+      var _inds = inds;
+      bulkAdd_help(_inds, isSorted, isUnique); 
+    }
+    else {
+      bulkAdd_help(inds, isSorted, isUnique);
+    }
+  }
+
+  // Tried to put this record in the function and the if statement, but got a
+  // segfault from the compiler.
```

Comment:
I think I want this record to be visible to users, but that doesn't have to fully happen in this PR. The reason is that if I'm calling myArray.bulkAdd, and I want to sort the data in-place in a separate step, how do I know what to sort by?